_signal_processing/krakenSDR_signal_processor.py

```diff
@@ -239,6 +241,46 @@ def resetPeakHold(self):
         if self.spectrum_fig_type == "Single":
             self.peak_hold_spectrum = np.ones(self.spectrum_window_size) * -200
 
+    def calculate_squelch(self, sampling_freq, N):
+        def is_enabled_auto_squelch(v):
```

Comment:
@redradist please move these functions out as we might need to wrap this up in Numba

Reply:
Did partially in last commit, but currently it is working pretty fast, any other method with jit-s only slows down it